apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,43 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
+  // Hand one is a variable that will just have the word Rock Paper or Scissor in it!
+
+  //Spec 1 - If it's a tie, return "It's a tie!" It must be exactly "It's a tie!" for the tests to pass
+
 
 function rockPaperScissors(hand1, hand2) {
+  if (hand1.toLowerCase() === hand2.toLowerCase()) {
```

Comment:
For this area I'm curious as to what would happen if you typed in two things that weren't rock, paper, or scissors? Like 'apple' & 'apple'?